test/Feature/arithmetic-right-overshift-sym-conc.c

```diff
@@ -13,14 +13,17 @@ typedef enum
     UNKNOWN
 } overshift_t;
 
+#define INT_31_BITS	(0xffffffff >> 1)
+
 // We're using signed ints so should be doing
 // arithmetic right shift.
```

Comment:
Hi @jirislaby , this makes sense to me, thanks.  But did you check you are still generating ashr after changing the types to `unsigned`?    At the very least, this comment should be updated, as it's confusing now.

Reply:
I rebased the patch and undid the `signed` -> `unsigned`. Now it works as expected and generates `ashr` too.